components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java

```diff
@@ -105,6 +103,9 @@ public void init(HazelcastInstance hazelcastInstance) {
             @Override
             public void memberAdded(MembershipEvent membershipEvent) {
                 checkMemberUpdate();
+                if(haManager!=null){
```

Comment:
Thanks @keizer619, Done in pull request - https://github.com/wso2/carbon-analytics-common/pull/109